tests/cypress/integration/actions_tasks2/case_101_opencv_basic_actions.js

```diff
@@ -0,0 +1,128 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName, labelName } from '../../support/const';
+
+context('OpenCV. Intelligent cissors. Histogram Equalization.', () => {
+    const caseId = '101';
+    const createOpencvShape = {
+        labelName: labelName,
+        pointsMap: [
+            { x: 200, y: 200 },
+            { x: 250, y: 200 },
+            { x: 300, y: 250 },
+            { x: 350, y: 300 },
+            { x: 300, y: 350 },
+        ],
+    };
+    const newLabel = `${createOpencvShape.labelName} new`
+    const createOpencvShapeSecondLabel = {
+        labelName: newLabel,
+        pointsMap: [
+            { x: 300, y: 200 },
+            { x: 350, y: 200 },
+            { x: 400, y: 250 },
+            { x: 450, y: 300 },
+            { x: 400, y: 350 },
+        ],
+        useDoneButton: true,
+    };
+    const keyCodeN = 78;
+    const pointsMap = [
+        { x: 300, y: 400 },
+        { x: 350, y: 500 },
+        { x: 400, y: 450 },
+        { x: 450, y: 500 },
+        { x: 400, y: 550 },
+    ];
+
+    function generateString(countPointsToMove) {
+        let action = '';
+        for (let i = 0; i < countPointsToMove; i++) {
+            action += '{rightarrow}';
+        }
+        return action;
```

Comment:
This function is already defined three times in ``tests/cypress/integration/**/*.ts`` (this is the 4th).
Maybe put it to ``utils.js`` file?

Reply:
Done. The corresponding tests are adapted for this.